app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  height: 30px;
+  background-color: rgba(0,0,0,0.75);
+  color: #fff;
+  padding: 5px;
+
+  .resources {
+    float: left;
+  }
+
+  .user-actions {
+    float: right;
+    a {
+      color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.